library/panic_unwind/src/gcc.rs

```diff
@@ -123,6 +123,9 @@ const UNWIND_DATA_REG: (i32, i32) = (0, 1); // R0, R1
 #[cfg(target_arch = "riscv64")]
 const UNWIND_DATA_REG: (i32, i32) = (10, 11); // x10, x11
 
+#[cfg(target_arch = "riscv32")]
+const UNWIND_DATA_REG: (i32, i32) = (10, 11); // x10, x11
```

Comment:
nit: these could use one `cfg`.
```suggestion
#[cfg(any(target_arch = "riscv64", target_arch = "riscv32"))]
const UNWIND_DATA_REG: (i32, i32) = (10, 11); // x10, x11
```

Reply:
Done